exampleSite/content/docs/faq.md

```diff
@@ -17,12 +17,17 @@ in the documentation.
 
 <!--more-->
 
-The answers have been categorized into two groups. Those are answers to general questions without any lines of code, and
-those are answers to technical questions with code snippets, step-by-step instructions, etc.
+The answers have been categorized into two groups:
+
+
+1. Answers to general questions without any lines of code.
+
+
```

Comment:
Extra empty lines

Reply:
yes this is for new line and give gap between the two
if there if error in hosted link then let me know